lib/perl/Genome/Model/Tools/Annotate/TranscriptVariants.pm

```diff
@@ -768,11 +768,8 @@ sub file_has_header {
 
     my @expected_column_headers = $class->variant_attributes();
     my $expected_header = join("\t", @expected_column_headers);
-    if ($first_line =~ m/^$expected_header/) {
-        return 1;
-    }
-    else {
-        return 0;
+    unless ($first_line =~ m/^$expected_header/) {
```

Comment:
[This isn't something you changed, but] since `$expected_header` is a constant this could be written as `unless (index($first_line, $expected_header) == 0) {`.